contracts/IVaultHandler.sol

```diff
@@ -638,13 +656,13 @@ abstract contract IVaultHandler is
    * @param _amount uint amount to mint
    * @return collateral of the TCAP Token
    * @dev TCAP token is 18 decimals
-   * @dev C = ((P * A * r) / 100) / cp
+   * @dev C = ((P * A * r) / 100) / (cp * cdaf)
    * C = Required Collateral
    * P = TCAP Token Price
    * A = Amount to Mint
    * cp = Collateral Price
    * r = Minimum Ratio for Liquidation
-   * Is only divided by 100 as eth price comes in wei to cancel the additional 0s
```

Comment:
This comment didn't seem correct to me.
100 is divided here to convert the `ratio` in percentage to a fraction.